README.md

```diff
@@ -1,11 +1,29 @@
 # libsir
 
+libsir &mdash; a cross-platform, thread-safe logging library
+
 <!-- SPDX-License-Identifier: MIT -->
 <!-- Copyright (c) 2018-current Ryan M. Lederman <lederman@gmail.com> -->
 
-[![Build Status](https://app.travis-ci.com/aremmell/libsir.svg?branch=master)](https://app.travis-ci.com/aremmell/libsir) ![REUSE Compliance](https://img.shields.io/reuse/compliance/github.com%2Faremmell%2Flibsir?label=REUSE3&color=2340b911)
+ [![Build Status](https://app.travis-ci.com/aremmell/libsir.svg?branch=master)](https://app.travis-ci.com/aremmell/libsir) ![REUSE Compliance](https://img.shields.io/reuse/compliance/github.com%2Faremmell%2Flibsir?label=REUSE3&color=2340b911)
```

Comment:
What effect does this have?

Reply:
Just an oversight, I'll fix it.  I'll actually use a non-breaking space to separate, although it won't really affect rendering until you have a whole line of badges lol.